apps/block_scout_web/assets/css/theme/_polygon_hermez_variables.scss

```diff
@@ -0,0 +1,194 @@
+@font-face {
+  font-family: "Modern Era";
+  src: url("../static/fonts/modern-era/ModernEra-Regular.woff2") format("woff2"),
+    url("../static/fonts/modern-era/ModernEra-Regular.woff") format("woff"),
+    url("../static/fonts/modern-era/ModernEra-Regular.ttf") format("truetype"),
+    local("Helvetica Neue"), local("Arial");
```

Comment:
Why do we need the Helvetica Neue and Arial fonts?

Reply:
These are fallbacks in case the download of the font fails. I thought it was better to use Arial than the default Times New Roman.

Reply:
But doing this we are saying that the `Modern Era` font is really a `Helvetica Neue` or `Arial` font. I think that it makes more sense to add this fallback to the `$font-family` variable.

```css
$font-family: "Modern Era", "Helvetica Neue", Arial;
```

Reply:
Right! I've tested it and works beautifully!

![image](https://user-images.githubusercontent.com/15896177/154996487-fb007e99-be33-44b0-9e7a-0d6cdac818de.png)